pkgs/applications/window-managers/cage/default.nix

```diff
@@ -1,19 +1,25 @@
-{ stdenv, fetchFromGitHub
+{ stdenv, fetchFromGitHub, fetchpatch
 , meson, ninja, pkgconfig, makeWrapper
 , wlroots, wayland, wayland-protocols, pixman, libxkbcommon
 , systemd, libGL, libX11
 , xwayland ? null
 }:
 
 stdenv.mkDerivation rec {
-  pname = "cage";
-  version = "0.1.1";
+  pname = "cage-unstable";
+  version = "2020-01-18";
```

Comment:
I believe the convention is to have the pname stay the same, and change the version to "unstable-yyyy-mm-dd"

Reply:
That's actually a common misconception, see: https://github.com/NixOS/nixpkgs/issues/68518

But it does seem a bit strange and there are arguments for both variants (though I also tend towards `pname` as version should start with a digit).

Reply:
Oh, that's kinda counter-intuitive, but sure